shared/xccdf/services/obsolete.xml

```diff
@@ -180,11 +218,13 @@ Enterprise Linux.</rationale>
 allow cleartext remote access and have an insecure trust
 model.</description>
 
-<Rule id="package_rsh-server_removed" severity="high" prodtype="rhel7">
+<Rule id="package_rsh-server_removed" severity="high" prodtype="rhel7,debian8,ubuntu1404">
 <title>Uninstall rsh-server Package</title>
 <description>The <tt>rsh-server</tt> package can be uninstalled with
```

Comment:
Please undo the description changes here. Use 2 separate descriptions with prodtype that makes the target os.